python/ray/serve/handle.py

```diff
@@ -134,6 +108,7 @@ def __init__(
         *,
         handle_options: Optional[_DynamicHandleOptionsBase] = None,
         _router: Optional[Router] = None,
+        _create_router: Optional[CreateRouterCallable] = None,
```

Comment:
Is this going to be used for local testing mode?

Reply:
yes